src/instructions/proctored_exam/download-instructions/index.jsx

```diff
@@ -50,21 +56,46 @@ const DownloadSoftwareProctoredExamInstructions = ({ intl, skipProctoredExam })
           setSystemCheckStatus('success');
         } else {
           softwareDownloadAttempt(attemptId);
-          window.open(downloadUrl, '_blank');
+          window.open(launchSoftwareUrl, '_blank');
```

Comment:
Question: this is the logic that opens in a new window right (per the ac in the ticket)?

Reply:
yup